libbeat/cmd/instance/beat.go

```diff
@@ -105,6 +105,7 @@ type beatConfig struct {
 
 	// beat internal components configurations
 	HTTP            *common.Config         `config:"http"`
+	HTTPPprof       *common.Config         `config:"http.pprof"`
```

Comment:
At first I was thinking this would conflict with the HTTP config one line before but it seems we have the same pattern on line 110/111?

Reply:
yeah, i followed the same pattern